public/app/features/explore/utils/ResultProcessor.test.ts

```diff
@@ -176,6 +176,7 @@ describe('ResultProcessor', () => {
               raw: 'third',
               searchWords: [] as string[],
               timeEpochMs: 300,
+              timeEpochNs: '300000000',
```

Comment:
Can we add here a case to check the sorting if the difference is sub milisecond to test the new behavior?

Reply:
@aocenas done, changed the tests slightly to address nanosecond-level precision.